### ISSUE TITLE
Error when locking in

### ISSUE THREAD

**Issue by NGPriest** [freyta/7Eleven-Python#4]
```[2019-03-16 12:31:54,129] ERROR in app: Exception on /lockin [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "app.py", line 483, in lockin
    session['SuccessMessage'] = "The price was locked in for " + str(session['LockinPrice']) + " cents per litre"
  File "/usr/local/lib/python2.7/dist-packages/werkzeug/local.py", line 377, in <lambda>
    __getitem__ = lambda x, i: x._get_current_object()[i]
  File "/usr/local/lib/python2.7/dist-packages/flask/sessions.py", line 83, in __getitem__
    return super(SecureCookieSession, self).__getitem__(key)
KeyError: 'LockinPrice'```

**Comment by alvinnguyen** [comment]
Make sure you got the API key has GeoCoding enable

**Comment by NGPriest** [comment]
Looks like it's been fixed with f8dddafcc0979d67ff40fb99c477347e7408bcc5

**Comment by freyta** [comment]
Yep, issue was fixed with the latest update. It was because the session variable "LockinPrice" was being popped (deleted) as soon as it locked the price, not before the redirect to the index page.
